README.md

```diff
@@ -215,11 +215,23 @@ SemVer getVersion()
 ```
 Retrives the node software version.
 
-- `getNodeInfo()`
+- `getNodeInfo`
 ```java
 NodeInfo getNodeInfo()
 ```
 
+- `banNode`
+```java
+boolean banNode(final BanNodeRequest request) throws Exception 
+```
+Ban a specific node.
+
+- `unBanNode`
+```java
+boolean unBanNode(final InetAddress ip)
```

Comment:
Please document what the `boolean` return value means.

Reply:
done